src/main/webapp/Profile.jsp

```diff
@@ -0,0 +1,242 @@
+<%@page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" %>
+
+  <%@page import="java.sql.*" %>
+    <%@page import="java.sql.Connection" %>
+      <%@page import="java.sql.DriverManager" %>
+        <%@page import="java.sql.ResultSet" %>
+          <%@page import="util.DBConnection" %>
+            <%@page import="javax.servlet.http.*" %>
+
+
+
+              <!DOCTYPE html>
+              <html lang="en">
+
+              <head>
+                <meta charset="UTF-8">
+                <meta http-equiv="X-UA-Compatible" content="IE=edge">
+                <meta name="viewport" content="width=device-width, initial-scale=1.0">
+                <title>Profile</title>
+                <link rel="stylesheet" href="profile.css">
+                <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+                  integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+                  crossorigin="anonymous">
+                   <script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"></script>
+
+              </head>
+
+              <body>
+                <!-- Nav Bar -->
+                <nav class="navbar navbar-dark bg-dark">
+                  <div class="container">
+                    <span class="navbar-brand mb-0 h1">Profile Page </span>
+                    <div>
+                      <a href="search.jsp" class="button">Search</a>
+                  </div>
+                      <div>
+                      <a href="logout" class="button">Log-Out</a>
+                  </div>
+                  </div>
+                </nav>
+
+
+                <div class="col-lg-8 mx-auto p-3 py-md-5">
+
+
+                  <header class="d-flex align-items-center pb-3 mb-5 border-bottom">
+                    <span class="fs-4">Welcome to your Profile</span>
+                    <br>
+                  </header>
+                  <% String username=(String) session.getAttribute("username"); String password=(String)
```

Comment:
This is a bad way of doing something. Please don't write too much Java Specific code in the JSP file. And code is also not fomatted.